src/main/java/mvs/model/Post.java

```diff
@@ -0,0 +1,95 @@
+package mvs.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by quangminh on 28/07/2017.
+ */
```

Comment:
Check lại toàn bộ các file entity nhá. 
1. Không thống nhất: item có/không private
2. khai báo @Table cho entity
3. Khai báo column